system/HTTP/DownloadResponse.php

```diff
@@ -128,6 +132,8 @@ public function getContentLength(): int
 
     /**
      * Set content type by guessing mime type from file extension
+     *
+     * @return void
      */
     private function setContentTypeByMimeType()
```

Comment:
```suggestion
     */
    private function setContentTypeByMimeType(): void
```
see https://github.com/codeigniter4/CodeIgniter4/pull/7794#discussion_r1286958733

Reply:
Ok! Done.